tests/addressbook/external_test.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (C) 2017  Lucas Hoffmann
+# This file is released under the GNU GPL, version 3 or a later revision.
+# For further details see the COPYING file
+from __future__ import absolute_import
+
+import unittest
+
+import mock
+
+from alot.addressbook import external
+
+
+class TestExternalAddressbookGetContacts(unittest.TestCase):
+
+    """Some test cases for
+    alot.addressbook.external.ExternalAddressbook.get_contacts"""
+
+    regex = '(?P<name>.*)\t(?P<email>.*)'
```

Comment:
Why? Does it matter if that is a literal tab or the two byte sequence `\t`? And there are not many backslashes in that string so `r'...'` doesn't make it any more readable.